src/stories/FormLabel.stories.tsx

```
@@ -0,0 +1,37 @@
+import { Meta } from '@storybook/react';
+
+const meta: Meta<typeof FormLabel> = {
+  title: 'FormLabel',
+  component: FormLabel,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+export const Default = (args: FormLabelProps) => <FormLabel {...args} />;
+export const Error = (args: FormLabelProps) => <FormLabel {...args} />;
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)